app/api/routes.py

```diff
@@ -914,6 +920,32 @@
     '/user-favourite-events/<int:id>/relationships/event',
 )
 
+# user favourite sessions
+api.route(
+    UserFavouriteSessionListPost, 'user_favourite_session_post', '/user-favourite-session'
+)
+api.route(
+    UserFavouriteSessionList,
+    'user_favourite_session_list',
+    '/user-favourite-session',
+    '/users/<int:user_id>/user-favourite-session',
```

Comment:
/users/<id>/favourite-sessions

Reply:
Another routes which need to be present:

/event/<id>/favourite-sessions: Returns favourite sessions of user in that event
/session/<id>/favourite-sessions: Returns all favourites of a session